app/fruits/page.tsx

```diff
@@ -46,7 +41,7 @@ export default function FruitsPage() {
             <Link href={`/fruits/${fruit.name.toLocaleLowerCase()}`}>
               <h2>{fruit.name}</h2>
               <p>{fruit.icon}</p>
-              <p>stars: {fruit.stars}</p>
+              <p>{fruit.comment || 'n/a'}</p>
```

Comment:
this `'n/a'` here looks a bit unusual from UI/UX perspective without any other title or label or anything

probably leaving it blank is best? (no default)

Reply:
c553de8

Reply:
I meant no default case (no `|| ''`)

Reply:
b185b14